specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2023-02-01-preview/purviewcatalog.json

```diff
@@ -7969,6 +7975,14 @@
       "in": "path",
       "x-ms-skip-url-encoding": true
     },
+    "Content-Type": {
+      "name": "Content-Type",
+      "description": "The name of the HTTP header field used to specify the media type. Typically values are 'applicaiton/json' and 'multipart/form-data'.",
```

Comment:
The description should indicate what customers can send to the Purview service and not the general HTTP description of this header. I suspect Purview ONLY ACCEPTS application/json (fix your spelling), right?